### PR TITLE
Use Android deploy script

### PR DESCRIPTION
**Pull request by matthewbauer**
Use deploy script provided by reflex-platform. This is linked to https://github.com/reflex-frp/reflex-platform/pull/619.

This avoids putting secrets into the Nix store.

I have:

  - [x] Based work on latest `develop` branch
  - [x] Looked for lint in my changes with `hlint .` (lint found code you did not write can be left alone)
  - [ ] Run the test suite: `$(nix-build -A selftest --no-out-link)`
  - [ ] (Optional) Run CI tests locally: `nix-build release.nix -A build.x86_64-linux --no-out-link` (or `x86_64-darwin` on macOS)

### CODE REVIEW

**Comment by 3noch** [comment]
@matthewbauer Sorry I had already fixed the nix-shell thing in #642

**Comment by 3noch** [comment]
Please merge in `develop`

File: dep/reflex-platform/github.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "owner": "reflex-frp",
   "repo": "reflex-platform",
-  "branch": "master",
+  "branch": "android-allow-post-build-signing",
```
Comment (danbornside) [inline dep/reflex-platform/github.json]: Not sure it makes sense to depend on a branch of reflex-platform;  lets get this merged there, too.

**Comment by 3noch** [comment]
We will enforce a released reflex-platform before releasing.

**Pull request merged**